Rover/Parameters.h

```diff
@@ -438,6 +438,8 @@ class ParametersG2 {
 
     // FS GCS timeout trigger time
     AP_Float fs_gcs_timeout;
+
+    class ModeCircle mode_circle;
```

Comment:
why belong to g2 ?

Reply:
@khancyr,

Thanks for looking over the PR.  g2 is where we've been putting most new objects in Rover.  SmartRTL and Follow mode area also in g2.